tools/run_tests/report_utils.py

```diff
@@ -118,3 +120,48 @@ def render_interop_html_report(
     print(exceptions.text_error_template().render())
     raise
 
+
+def render_perf_html_report(report_dir):
+  """Generate a simple HTML report for the perf tests."""
+  template_file = 'tools/run_tests/perf_html_report.template'
+  try:
+    mytemplate = Template(filename=template_file, format_exceptions=True)
+  except NameError:
+    print 'Mako template is not installed. Skipping HTML report generation.'
+    return
+  except IOError as e:
+    print 'Failed to find the template %s: %s' % (template_file, e)
+    return
+
+  resultset = {}
+  langs = []
+  for result_file in glob.glob(os.path.join(report_dir, '*.json')):
+    with open(result_file, 'r') as f:
+      scenario_result = json.loads(f.read())
+      name = scenario_result['scenario']['name']
+      divider = name.find('_')
+      lang = name[:divider]
+      if lang not in langs:
+        langs.append(lang)
+      test_case = name[divider+1:]
+      if 'ping_pong' in test_case:
+        latency50 = scenario_result['summary']['latency50']
```

Comment:
rounding the numbers and converting to microsec would make the result more readable (and also displaying the unit in the report).